gosearch.go

```
@@ -746,13 +747,25 @@ func MakeRequestWithErrorMsg(website Website, url string, username string) {
 	if err != nil {
 		return
 	}
+	var reader io.ReadCloser
+
+	switch res.Header.Get("Content-Encoding") {
+	case "gzip":
+		reader, err = gzip.NewReader(res.Body)
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)